conn_test.go

```diff
@@ -135,29 +151,33 @@ func TestGrpcConnection_Close(t *testing.T) {
 	wg.Wait()
 }
 
-func TestConnectionPool_Broadcast(t *testing.T) {
+func TestConnectionPool_BroadcastMessage(t *testing.T) {
 	cp := cleisthenes.NewConnectionPool()
 
-	cAddress1 := cleisthenes.Address{"127.0.0.1", 8080}
-	cAddress2 := cleisthenes.Address{"127.0.0.1", 8081}
+	cAddressList := make([]cleisthenes.Address, 3)
 
-	mockStreamWrapper1 := mock.NewStreamWrapper()
-	conn1, err1 := cleisthenes.NewConnection(cAddress1, "TestConnection", mockStreamWrapper1)
-	if err1 != nil {
-		t.Fatal(err1)
+	for i, _ := range cAddressList {
+		cAddressList[i] = cleisthenes.Address{"127.0.0.1", GetAvailablePort(8000)}
 	}
 
-	mockStreamWrapper2 := mock.NewStreamWrapper()
-	conn2, err2 := cleisthenes.NewConnection(cAddress2, "TestConnection2", mockStreamWrapper2)
-	if err2 != nil {
-		t.Fatal(err2)
+	mockStreamWrapperList := make([]*mock.StreamWrapper, 3)
+	connList := make([]cleisthenes.Connection, 3)
+	var err error
```

Comment:
what is it?

Reply:
I made it because I need error variable for get return value at 168 line.